### DISCUSSION TITLE
Remix + SASS + Tailwind ?

### DISCUSSION THREAD

**Discussion by jocieldo** [remix-run/remix#3609] [General]
Hey guys, just to ask if it's possible configure the remix with sass and tailwind together, if the answer is yes, can you please give me some idea how to do it? Thank you so much.

**Comment by sergiodxa** [comment]
Yes, you need to run a process to generate the Tailwind CSS file your Remix app imports, and another process to compile your SASS code to CSS files your Remix app imports.

Because you had to run both as standalone processes there’s nothing specific of Remix, run then and import the final CSS files those processes output.
